packages/react-native-editor/metro.config.js

```diff
@@ -13,7 +13,7 @@ const packageNames = fs.readdirSync( PACKAGES_DIR ).filter( ( file ) => {
 module.exports = {
 	watchFolders: [ path.resolve( __dirname, '../..' ) ],
 	resolver: {
-		sourceExts: [ 'js', 'json', 'scss', 'sass', 'ts', 'tsx' ],
+		sourceExts: [ 'js', 'cjs', 'json', 'scss', 'sass', 'ts', 'tsx' ],
```

Comment:
Attempt to fix this error that happens after updating `rememo`:

```sh
Error: While trying to resolve module `rememo` from file `/Users/runner/work/gutenberg/gutenberg/packages/blocks/src/store/selectors.js`, the package `/Users/runner/work/gutenberg/gutenberg/node_modules/rememo/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/Users/runner/work/gutenberg/gutenberg/node_modules/rememo/rememo.cjs`. Indeed, none of these files exist:
  * /Users/runner/work/gutenberg/gutenberg/node_modules/rememo/rememo.cjs(.native|.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.scss|.native.scss|.scss|.ios.sass|.native.sass|.sass|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)
```

Reply:
It seems to be causing an issue with the react native build, unfortunately. I'll have to see if we need to do anything else to fix it